### PR TITLE
Remove Doug from codeowners, area-owners

### PR DESCRIPTION
**Pull request by wtgodbe**
Side note, @mkArtakMSFT should we replace Tanay with somebody else where he appears in CODEOWNERS?

### CODE REVIEW

**Review by captainsafia — APPROVED** [review]

**Review by dougbu — APPROVED** [review]
ta

**Comment by dougbu** [comment]
@wtgodbe, please also clear things up in servicing branches, aspnetcore-internal, and efcore.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
-/global.json                                                                      @dotnet/aspnet-build @dougbu @wtgodbe
-/.azure/                                                                          @dotnet/aspnet-build @dougbu @wtgodbe
-/.azuredevops/                                                                    @dotnet/aspnet-build @dougbu @wtgodbe
-/.config/                                                                         @dotnet/aspnet-build @dougbu @wtgodbe
+/global.json                                                                      @dotnet/aspnet-build @wtgodbe
+/.azure/                                                                          @dotnet/aspnet-build @wtgodbe
+/.azuredevops/                                                                    @dotnet/aspnet-build @wtgodbe
+/.config/                                                                         @dotnet/aspnet-build @wtgodbe
 /.devcontainer/                                                                   @captainsafia @TanayParikh
 /.vscode/                                                                         @captainsafia @TanayParikh
```
Comment (mkArtakMSFT) [inline .github/CODEOWNERS:10]:
```suggestion
/.devcontainer/                                                                   @captainsafia
/.vscode/                                                                         @captainsafia
```

**Comment by dougbu** [comment]
> > please also clear things up in servicing branches, aspnetcore-internal, and efcore.
> 
> &hellip; Aspnetcore-internal doesn't mention you currently

See docs\engineering\SMEs.md and docs\operations\contacts.md

> I'll get PRs up for servicing branches as well

👍

**Review by mkArtakMSFT — APPROVED** [review]
Thanks @wtgodbe !

**Pull request merged**